core/admin/mailu/models.py

```diff
@@ -589,6 +612,25 @@ def check_password(self, password):
         """
         if password == '':
             return False
+        
+        if utils.oic_client.is_enabled():
+            if 'openid_token' not in session:
+                try:
+                    openid_token = utils.oic_client.get_token(self.email, password)
+                    if openid_token is None:
+                        return self.check_password_legacy(password)
+                    session['openid_token'] = openid_token
+                except: 
+                    return self.check_password_legacy(password)
```

Comment:
I am taking offense with the terminology used here. "legacy" is not appropriate to describe the current authentication flow.

Reply:
I took it as it was in the fork, will rename it :wink: